Formula/metanorma.rb

```diff
@@ -1,36 +1,40 @@
 # frozen_string_literal: true
 
-require "language/node"
 require "language/python"
 
-class Metanorma < Formula
+# https://github.com/metanorma/packed-mn
+class Metanorma < Formula # rubocop:disable Metrics/ClassLength
   include Language::Python::Virtualenv
 
   desc "Toolchain for publishing metanorma documentation"
   homepage "https://www.metanorma.com"
 
   # > formula-set-version.sh #
-  url "https://github.com/metanorma/metanorma-cli/archive/v1.3.7.1.tar.gz"
-  sha256 "e08cd9a79f15c0c2bacbd603811eeb1a21001b3440d2c381190cec680e14d949"
+  url "https://github.com/metanorma/packed-mn/archive/v1.3.7.3.tar.gz"
+  sha256 "f281f19fc3631487a075244eb038f8cf1fa8b4d65e54293fc740886cf01a7b95"
   # < formula-set-version.sh #
 
   license "0BSD"
+  revision 1
 
   depends_on "latexml"
```

Comment:
Since we're using packed-mn, we can remove latexml, plantuml and python, right?

Reply:
I think no because we packed only ruby code to a native binary. metanorma still continue to use latexml, plantuml and xml2rfc as standalone tools

Reply:
Oh right. I got confused. But we should be able to remove latexml later because we don’t depend on it for latex processing (subject to double checking).